From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Amaan Qureshi <git@amaanq.com>
Date: Sat, 11 Jan 2026 00:00:00 +0000
Subject: [PATCH] Increment version for Kagi addition

Increment search engine data version to ensure Kagi appears in
users' search engine lists.
---

diff --git a/definitions/prepopulated_engines.json b/definitions/prepopulated_engines.json
--- a/definitions/prepopulated_engines.json
+++ b/definitions/prepopulated_engines.json
@@ -28,7 +28,7 @@
     // existing data should get a new version. Otherwise, existing data may
     // continue to be used and updates made here will not always appear.
     // Also then run tools/search_engine_choice/generate_search_engine_icons.py.
-    "kCurrentDataVersion": 193
+    "kCurrentDataVersion": 194
   },
 
   // The following engines are included in country lists and are added to the
